schematic_api/api/routes.py

```diff
@@ -43,77 +31,10 @@
 logger = logging.getLogger(__name__)
 logging.basicConfig(level=logging.DEBUG)
 
-tracing_service_name = os.environ.get("TRACING_SERVICE_NAME", "schematic-api")
-
-trace.set_tracer_provider(
-    TracerProvider(resource=Resource(attributes={SERVICE_NAME: tracing_service_name}))
-)
-
-
-# borrowed from: https://github.com/Sage-Bionetworks/synapsePythonClient/blob/develop/tests/integration/conftest.py
-class FileSpanExporter(ConsoleSpanExporter):
-    """Create an exporter for OTEL data to a file."""
-
-    def __init__(self, file_path: str) -> None:
-        """Init with a path."""
-        self.file_path = file_path
-
-    def export(self, spans: List[Span]) -> None:
-        """Export the spans to the file."""
-        with open(self.file_path, "a", encoding="utf-8") as f:
-            for span in spans:
-                span_json_one_line = span.to_json().replace("\n", "") + "\n"
-                f.write(span_json_one_line)
-
-
-def set_up_tracing() -> None:
-    """Set up tracing for the API."""
-    tracing_export = os.environ.get("TRACING_EXPORT_FORMAT", None)
-    if tracing_export == "otlp":
-        trace.get_tracer_provider().add_span_processor(
-            BatchSpanProcessor(OTLPSpanExporter())
-        )
-    elif tracing_export == "file":
-        timestamp_millis = int(time.time() * 1000)
-        file_name = f"otel_spans_integration_testing_{timestamp_millis}.ndjson"
-        file_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), file_name)
-        processor = SimpleSpanProcessor(FileSpanExporter(file_path))
-        trace.get_tracer_provider().add_span_processor(processor)
-    else:
-        trace.set_tracer_provider(TracerProvider(sampler=ALWAYS_OFF))
-
 
-set_up_tracing()
 tracer = trace.get_tracer("Schematic")
 
 
-def trace_function_params():
-    """capture all the parameters of API requests"""
-
-    def decorator(func):
-        """create a decorator"""
-
-        @wraps(func)
-        def wrapper(*args, **kwargs):
-            """create a wrapper function. Any number of positional arguments and keyword arguments can be passed here."""
-            tracer = trace.get_tracer(__name__)
-            # Start a new span with the function's name
-            with tracer.start_as_current_span(func.__name__) as span:
-                # Set values of parameters as tags
-                for i, arg in enumerate(args):
-                    span.set_attribute(f"arg{i}", arg)
-
-                for name, value in kwargs.items():
-                    span.set_attribute(name, value)
-                # Call the actual function
-                result = func(*args, **kwargs)
-                return result
-
-        return wrapper
-
-    return decorator
-
-
```

Comment:
This is no longer needed with the request hooks that I added for the `FlaskInstrumentor`